services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/attribute_monitor/_logging_event_handler.py

```diff
@@ -0,0 +1,226 @@
+# pylint:disable=no-member
+
+import logging
+import stat
+from asyncio import CancelledError, Task, create_task, get_event_loop
+from asyncio import sleep as async_sleep
+from concurrent.futures import ThreadPoolExecutor
+from contextlib import suppress
+from pathlib import Path
+from queue import Empty
+from time import sleep as blocking_sleep
+from typing import Final, Optional
+
+import aioprocessing
+from aioprocessing.process import AioProcess
+from aioprocessing.queues import AioQueue
+from pydantic import ByteSize, PositiveFloat
+from servicelib.logging_utils import log_context
+from watchdog.events import FileSystemEvent
+
+from ._watchdog_extensions import ExtendedInotifyObserver, SafeFileSystemEventHandler
+
+_HEART_BEAT_MARK: Final = 1
+
+logger = logging.getLogger(__name__)
+
+
+class _LoggingEventHandler(SafeFileSystemEventHandler):
+    # NOTE: runs in the created process
+
+    def event_handler(self, event: FileSystemEvent) -> None:
+        file_path = Path(event.src_path)
+        file_stat = file_path.stat()
+        logger.info(
+            "Attribute change to: '%s': permissions=%s uid=%s gid=%s size=%s\nFile stat: %s",
+            file_path,
```

Comment:
I guess you cannot show from what to what it was changed right? (e.g. old permissions vs new permissions)